### ISSUE TITLE
Always the first series of the first season.

### ISSUE THREAD

**Issue by daznik** [Tommixoft/plugin.video.lostfilm.tv#3]
Hi, I have always only first season first series playing, doesn't matter which series i start. Anybody seen this behavior?

**Comment by Tommixoft** [comment]
Will try to look at it tomorrow. But so far nobody else complained.

**Comment by daznik** [comment]
Here the log, anything starts only from first season first episode.
https://pastebin.com/bQBATZpv
Let me know if any additional info will be helpfull.

**Comment by Tommixoft** [comment]
@daznik 
Hi i tested my installation and i can play any season and any episode.
Are you sure you are using my version of LostFilm plugin? Do you see menu item in lost film plugin named "**Latest Trailers**" ? or "**_100 Best rated ended tv shows_**" if no -you are not using this plugin.

**Comment by daznik** [comment]
Sure, I've browsed the "100 Best rated ended tv shows" . 
As can be seen on logs, request goes to different series : 
```
 [xbmcswift2] Request for "/play_episode/203/1/2" matches rule for function "play_episode"
21:17:57.351 T:1320674048  NOTICE: [lostfilm.network_request] Fetching URL http://www.lostfilm.tv/v_search.php?c=203&**s=1&e=2** with params None started... 
```

but it always saving thr first episode : 

`> `NOTICE: [support.torrent.stream.t2h_stream] Detected video file: FileStatus(name=u'Klondike 1 - LostFilm.TV [1080p]/Klondike.S01E01.1080p.rus.LostFilm.TV.mkv', save_path=u'/home/osmc/.kodi/temp/torrent2http/Klondike 1 - LostFilm.TV [1080p]/Klondike.S01E01.1080p.rus.LostFilm.TV.mkv', url=u'http://127.0.0.1:5001/files/Klondike%201%20-%20LostFilm.TV%20%5B1080p%5D/Klondike.S01E01.1080p.rus.LostFilm.TV.mkv', size=2179835602L, offset=0, download=0, progress=0, index=0, media_type='video')``


Maybe the problem with torrent2http?

**Comment by Tommixoft** [comment]
Hmm hard to tell where problem is. I think i would have it too if that
would e in plugin, or if LostFilm had changed anything. Also my mom uses it
and she have no problems too.
Maybe try to delete all temp files where torrent2http saves temp files?
What OS are you using? I only have Windows OS.

**Comment by daznik** [comment]
Linux,
Ok if it still works for you then there is some problems in my side. What version of plugin& torrent2http are you using?

**Comment by Tommixoft** [comment]
Oh i see from logs looks like you are using Android.  Android uses other binaries. But i think those were not updated so strange if problem happened from nothing. Try to clear temp directory of torrent engine. Or try to watch new tv show that you never watch and try to download some other then 1st episode. 
Also i see you are using OSMC and not Kodi. Maybe problem is in that? also my KODI is v16, so it's old, maybe problem is with newer version of Kodi. MY mom too uses old version of OSMC with Android. 
Maybe you updated OSMC?
Now it's late, will look tommorow at versions. but torrents2http is the same that is in this git.

**Comment by Jabberwacky** [comment]
> Hi, I have always only first season first series playing, doesn't matter which series i start. Anybody seen this behavior?

I mention this in a very old serisals. Such as "Star Trek" or "Firefly". In the last case I guess it is because the serial is a one huge media-file. Probably

**Comment by Tommixoft** [comment]
@Jabberwacky yeah it might be, i tested plugin on 3 different players and they all worked ok. So something else causes this.
